lpa-submissions-e2e-tests/cypress/integration/common/uploadFiles.js

```diff
@@ -204,6 +203,7 @@ Then('any document uploaded will not be saved', () => {
 });
 
 Then('the file is removed', () => {
+  cy.wait(1000);
```

Comment:
This line needs to be changed to be in line with standards.

Reply:
This is required as there is a ms difference in the file getting deleted and updated